conf/tasks/Makefile.kops

```diff
@@ -1,6 +1,13 @@
 SSH_KEY ?= $(CACHE_PATH)/id_rsa
 SSH_PUBLIC_KEY ?= $(SSH_KEY).pub
 
+## Prep the kiosk for cluster creation
+kops/prep:
+	@echo "Prepping the kiosk for cluster creation..."
+	apk add gettext
```

Comment:
`gettext` should already be in `geodesic`, but either way, move this to the `Dockerfile`

Reply:
`read` function is actually a bash internal.  `gettext` is for `envsubst`

Reply:
hrm... though there is a `/usr/bin/read` too... 😕